src/Symfony/Component/Validator/Mapping/GenericMetadata.php

```diff
@@ -38,8 +38,6 @@ class GenericMetadata implements MetadataInterface
     public array $constraints = [];
 
     /**
-     * @var array
```

Comment:
I suggest updating it to `array<string, list<Constraint>>` to have the precise type instead of removing the type.

Reply:
sure. automate dropped the pointless type hint. if you have better one, happy to add it.

also looking at other annotations around those properties, they use `Constraint[]`, let's be consistent on it - i followed what's already present
de34f51629eb7372a43217aa85510051fb7bcf17